62.CAD/common.hlsl

```diff
@@ -53,11 +98,21 @@ struct Globals
 
 struct LineStyle
 {
+    // common data
     float4 color;
     float screenSpaceLineWidth;
     float worldSpaceLineWidth;
-    // TODO[Przemek]: Anything info related to the stipple pattern will be here
-    float _pad[2u];
+    
+    // stipple pattern data
+    static const uint32_t STIPPLE_PATTERN_MAX_SZ = 15u;
```

Comment:
move the static stuff to the beginning. cause when I start looking at the struct I want to be able to guess the size reading top to botton

Reply:
done